src/rpc/handlers/LedgerData.h

```diff
@@ -42,6 +44,10 @@ class LedgerDataHandler
     static uint32_t constexpr LIMITBINARY = 2048;
     static uint32_t constexpr LIMITJSON = 256;
 
+    static const std::unordered_map<std::string, ripple::LedgerEntryType> TYPES_MAP;
+
+    static const std::vector<std::string> TYPES_KEYS;
```

Comment:
i don't mind vector but logically it probably should be a set? any reason to use vector instead?